nextcord/voice_client.py

```diff
@@ -574,6 +574,7 @@ def play(self, source: AudioSource, *, after: Callable[[Optional[Exception]], An
             The finalizer that is called after the stream is exhausted.
             This function must have a single parameter, ``error``, that
             denotes an optional exception that was raised during playing.
+            If the function is a coroutine, it will be awaited when called.
```

Comment:
Do we want to also change the types to include coros?

Reply:
What do you suggest?

If `Callable[[Optional[Exception]], Union[Coroutine, Any]]`: `Coroutine` is included in `Any`, so I don't think it's necessary to pass type checkers. It might be useful for humans, but that's what the docstring is for.

https://github.com/python/typing/issues/424 has some notes about this, though as discovered earlier we don't want a scope as large as `Awaitable`.

Reply:
Yeah it would have some form of Any for sync functions, so type checkers ignore it

Reply:
I'm not really sure what further action to take here.